docs/tutorial/blog-netlify-cms-tutorial/index.md

```diff
@@ -1,91 +1,91 @@
-gatsby new [your-project-name] https://github.com/thomaswangio/gatsby-personal-starter-blog
+gatsby new [nombre-de-tu-proyecto] https://github.com/thomaswangio/gatsby-personal-starter-blog
 ```
 
-### Step 2
+### Paso 2
 
-Once the Gatsby site is finished installing all the packages and dependencies, you can now go into the directory and run the site locally.
+Una vez que el sitio de Gatsby haya terminado de instalar todos los paquetes y dependencias, podrás ir al directorio y ejecutar el sitio localmente.
 
 ```shell
-cd [your-project-name]
+cd [nombre-de-tu-proyecto]
 gatsby develop
 ```
 
-Now you can go to [`localhost:8000`](http://localhost:8000) to see your new site, but what's extra cool is that Netlify CMS is pre-installed and you can access it at [`localhost:8000/admin`](http://localhost:8000/admin).
+Ahora puedes ir a [`localhost:8000`](http://localhost:8000) para ver tu nuevo sitio, pero lo que es genial es que Netlify CMS está preinstalado y puedes acceder a él en [`localhost:8000/admin`](http://localhost:8000/admin).
 
-A CMS, or content management system, is useful because you can add content like blog posts from a dashboard on your site, instead of having to add posts manually with Markdown. However, you'll likely want to be able to access the CMS from a deployed website, not just locally. For that, you'll need to deploy to Netlify through GitHub, set up continuous deployment, and do a few configurations. We'll go over this in [Step-5](#step-5).
+Un CMS, o sistema de administración de contenido, es útil porque puedes agregar contenido como publicaciones de blog desde un tablero en tu sitio, en lugar de tener que agregar publicaciones manualmente con Markdown. Sin embargo, es probable que desees poder acceder al CMS desde un sitio web desplegado, no solo localmente. Para eso deberás desplegar en Netlify a través de GitHub, preparar una implementación continua y realizar algunas configuraciones. Repasaremos esto en el [Paso-5](#step-5).
 
-### Step 3
+### Paso 3
 
-Open the project in your code editor and open `static/admin/config.yml`. Replace `your-username/your-repo-name` with your GitHub username and project name. This step is important for managing and deploying the Netlify CMS interface.
+Abre el proyecto en tu editor de código y luego el archivo `static/admin/config.yml`. Reemplaza `tu-nombre-de-usuario/tu-nombre-de-repositorio` con tu nombre de usuario y de proyecto de GitHub. Este paso es importante para administrar e implementar la interfaz de Netlify CMS.
 
 ```diff
 backend:
 -  name: test-repo
 
 +  name: github
-+  repo: your-username/your-repo-name
++  repo: tu-nombre-de-usuario/tu-nombre-de-repositorio
 ```
 
-#### Customizing your site
+#### Personalizando tu sitio
 
-Head into `gatsby-config.js` and you can edit your siteMedata, add a Google Analytics tracking ID, and your app icon/favicon. Test out the edits for the deployed build by quitting the development server and running `gatsby build && gatsby serve`.
```

Comment:
I found a typo in the official documentation that I corrected here. Should we report it?

siteMedata ❌  -->  siteMetadata ✅

Reply:
Yes, you should I just saw some Russians opening PRs from typos while they were translating muscle